package.json

```diff
@@ -25,22 +25,24 @@
     "radium": "0.19.1"
   },
   "devDependencies": {
```

Comment:
The version should be updated in this file.

Reply:
Upgraded to v0.25.2 which is the latest without breaking changes. No vulnerabilities were found.